package/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "brave-ui",
   "main": "index.js",
-  "version": "0.29.3",
```

Comment:
Was this meant to get updated in this PR @cezaraugusto ?

Reply:
oops changed